masschallenge/semantic-ui-theme
# [AC-6776] Fix horizontally scroll tab sub navigation on startup dashboard

**Changes introduced in [AC-6776](https://masschallenge.atlassian.net/browse/AC-6776)**
- Allow horizontal scrolling for sub navigation

**How to test**
- Masquerade as [user](http://localhost:8181/admin/simpleuser/user/43639/masquerade/)
- Go to the a startup [dashboard](http://localhost:8181/startup/9350/dashboard/)
- Reduce the size of the screen and you will see that the subnav menu doesn't overflow and that the dropdown also works.
